tasks.py

```diff
@@ -251,9 +251,12 @@ def validate_kind_version():
     "node_img": "Optional node image to use for the kind cluster (e.g. kindest/node:v1.18.19)."
                 "The node image drives the kubernetes version used in kind.",
     "ip_family": "Optional ipfamily of the cluster."
-                 "Default: ipv4, supported families are 'ipv6' and 'dual'."
+                 "Default: ipv4, supported families are 'ipv6' and 'dual'.",
+    "bgp_type": "Type of BGP implementation to use."
+                "Supported: 'native' (default), 'frr'"
```

Comment:
could have also been `-p bgp-native` and `-p bgp-frr` instead of a new option I guess, but this works too

Reply:
We want to be able to differentiate, because CI now doesn't care about the protocol.
Using -p will add a pre-canned configuration which will interfere with the test execution.

Reply:
good point, thanks